docs/index.md

```diff
@@ -1499,7 +1500,7 @@ To specify custom report title, use `--reporter-options suiteName="Custom name"`
 ### Third-Party Reporters
 
 Mocha allows you to define custom reporters. For more information see the [wiki][mocha-wiki-more-reporters].
-
+.equal
```

Comment:
a typo?

Reply:
Yup. Probably me typing in the incorrect place. Will fix.